### PR TITLE
use env urls

### PR DESCRIPTION
**Pull request by birdbrained**

### CODE REVIEW

File: client/config/environment.js [hunk 1]
```diff
@@ -34,6 +35,7 @@ module.exports = function(environment) {
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
 
+    ENV['osfHostUrl'] = 'http://staging-api.osf.io';
```
Comment (samanehsan) [inline client/config/environment.js]: Should this be here? `ENV['osfHostUrl']` is also set to `'http://staging-api.osf.io'` on line 56

File: client/config/environment.js [hunk 2]
```diff
@@ -50,7 +52,12 @@ module.exports = function(environment) {
     ENV.APP.rootElement = '#ember-testing';
   }
 
+  if (environment === 'stage') {
+    ENV['osfHostUrl'] = 'http://staging-api.osf.io';
```
Comment (samanehsan) [inline client/config/environment.js]: This url needs to be `'https://staging-api.osf.io'` and same with prod

File: client/config/environment.js (hunk 1 above)
Comment (samanehsan) [inline client/config/environment.js]: Oh nevermind, I didn't notice it was inside of an if statement

Comment (birdbrained) [inline client/config/environment.js]: ah

Comment (birdbrained) [inline client/config/environment.js]: oh wait yeah. this is in a conditional

File: client/config/environment.js (hunk 2 above)
Comment (birdbrained) [inline client/config/environment.js]: ah missing the s

**Pull request merged**